.composer-require-checker.json

```diff
@@ -0,0 +1,9 @@
+{
+    "symbol-whitelist": [
+        "PhpCsFixer\\PhpunitConstraintIsIdenticalString\\Constraint\\IsIdenticalString",
```

Comment:
why we need this?

Reply:
It is used in `class_alias` and recognised as missing dependency: https://github.com/maglnet/ComposerRequireChecker/issues/108